packages/create-frontity/README.md

```diff
@@ -0,0 +1,60 @@
+# `create-frontity`
+
+[![Version](https://img.shields.io/npm/v/create-frontity.svg)](https://www.npmjs.com/package/create-frontity) [![npm](https://img.shields.io/npm/dw/create-frontity)](https://www.npmjs.com/package/create-frontity) [![License: Apache--2.0](https://img.shields.io/badge/license-Apache%202-lightgrey)](https://github.com/frontity/frontity/blob/master/LICENSE)
+
+This package is linked to `npx frontity create`. 
+This way, people can use `npx frontity create` as well as `npm init frontity`.
+
+## Table of contents
+
+<!-- toc -->
+
+- [Install](#install)
+- [Feature Discussions](#feature-discussions)
+- [Changelog](#changelog)
+  * [» Frontity Resources 🌎](#%C2%BB-frontity-resources-%F0%9F%8C%8E)
+  * [» Get involved 🤗](#%C2%BB-get-involved-%F0%9F%A4%97)
+
+<!-- tocstop -->
+
+## Install
+
+```sh
+npm i create-frontity
+```
+
```

Comment:
This package doesn't need to be installed. It's meant to be used only in conjunction with the `npm init` command, like this:

```bash
> npm init frontity
```